### PR TITLE
Tabs: move role attr from root to wrapper to fix Axe error

### PR DESCRIPTION
**Pull request by Zedwag**

### CODE REVIEW

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.markup.tests.js [hunk 1]
```diff
@@ -129,14 +129,16 @@ QUnit.module('Aria accessibility', {
...
+                helper.checkAttributes(helper.widget._$wrapper, { role: 'tablist' }, 'tabsWrapper');
                 helper.checkItemsAttributes([], { attributes: ['aria-selected'], role: 'tab' });
             });
 
             QUnit.test(`[item1], add new item2, repaintChangesOnly: ${repaintChangesOnly}, use: ${sourceName}`, function(assert) {
                 helper.createWidget({ [`${sourceName}`]: [{ text: 'Item_1' }], repaintChangesOnly });
 
-                helper.checkAttributes(helper.$widget, { role: 'tablist', tabindex: '0' }, 'widget');
+                helper.checkAttributes(helper.$widget, { tabindex: '0' }, 'widget');
+                helper.checkAttributes(helper.widget._$wrapper, { role: 'tablist' }, 'tabsWrapper');
```
Comment (ksercs) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.markup.tests.js]: Can we get rid of using of a private API?

File: packages/devextreme/js/ui/tabs.js [hunk 2]
```diff
@@ -383,6 +382,7 @@ const Tabs = CollectionWidget.inherit({
 
     _renderWrapper: function() {
         this._$wrapper = $('<div>').addClass(TABS_WRAPPER_CLASS);
+        this.setAria('role', 'tablist', this._$wrapper);
```
Comment (ksercs) [inline packages/devextreme/js/ui/tabs.js:385]: is Screen reader UI still correct?

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.markup.tests.js [hunk 3]
```diff
@@ -147,38 +149,44 @@ QUnit.module('Aria accessibility', {
...
                 helper.widget.option('focusedElement', helper.getItems().eq(1));
-                helper.checkAttributes(helper.$widget, { role: 'tablist', 'aria-activedescendant': helper.widget.getFocusedItemId(), tabindex: '0' }, 'widget');
+                helper.checkAttributes(helper.$widget, { tabindex: '0' }, 'widget');
+                helper.checkAttributes(helper.widget._$wrapper, { role: 'tablist', 'aria-activedescendant': helper.widget.getFocusedItemId(), }, 'tabsWrapper');
                 helper.checkItemsAttributes([1], { focusedItemIndex: 1, attributes: ['aria-selected'], role: 'tab' });
 
                 helper.widget.option('focusedElement', null);
-                helper.checkAttributes(helper.$widget, { role: 'tablist', tabindex: '0' }, 'widget');
+                helper.checkAttributes(helper.$widget, { tabindex: '0' }, 'widget');
+                helper.checkAttributes(helper.widget._$wrapper, { role: 'tablist' }, 'tabsWrapper');
```
Comment (ksercs) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.markup.tests.js]: Let's additionally write a matrix testcafe test using AXE? [can be done in a separate task]

File: packages/devextreme/js/ui/tabs.js (hunk 2 above)
Comment (Zedwag) [reply packages/devextreme/js/ui/tabs.js:385]: yep

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.markup.tests.js (hunk 1 above)
Comment (Zedwag) [reply packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.markup.tests.js]: done

**Review by ksercs — APPROVED** [review]

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.markup.tests.js (hunk 3 above)
Comment (Zedwag) [reply packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.markup.tests.js]: we'll do it separetely

**Review by AlexanderMoiseev — APPROVED** [review]

**Review by marker-dao — APPROVED** [review]

**Pull request merged**